### PR TITLE
Document how the staging / integration sync works

### PR DESCRIPTION
**Pull request by 36degrees**
The GovDelivery topic mapping sync is a little confusing. This is an attempt to document it.

### CODE REVIEW

**Comment by tijmenb** [comment]
This is great. Could you put it in the main README for more visibility?

File: doc/integration-staging-sync.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+GovDelivery Topic IDs that we hold in our database continue to link to the
+correct topics in every environment.
+
+This is complicated by the fact that both integration and staging talk to the
+same staging 'instance' of GovDelivery. Despite being separate accounts, the
+GovDelivery topic ids have to be unique across the entire instance.
+
+To achieve the synchronisation we have a rake task
+(`sync_govdelivery_topic_mappings`) which is automatically triggered when the
+'Data Sync Complete' job runs in Jenkins.
```
Comment (tijmenb) [inline doc/integration-staging-sync.md]: Can you link to https://deploy.integration.publishing.service.gov.uk/job/Data_Sync_Complete/?

**Comment by 36degrees** [comment]
It seemed a bit much for the main README, which is why I didn't put it there originally. Would linking to it suffice, or do you think it should just live there?

File: doc/integration-staging-sync.md (hunk 1 above)
Comment (36degrees) [inline doc/integration-staging-sync.md]: I could, but it's run in staging as well. I could do something like

> …which is automatically triggered when the 'Data Sync Complete' job runs in Jenkins on [integration](https://deploy.integration.publishing.service.gov.uk/job/Data_Sync_Complete/) and [staging](https://deploy.staging.publishing.service.gov.uk/job/Data_Sync_Complete/).

?

**Comment by 36degrees** [comment]
@tijmenb would you mind re-reviewing this? Ta.

**Pull request merged**